plugins/modules/pnp_intent.py

```diff
@@ -786,15 +818,17 @@ class instance for further use.
                 )
 
                 self.have["deviceInfo"] = dev_add_response.get("deviceInfo")
-                self.log(str(dev_add_response))
+                self.log("Single device addition response from the API 'add device' is {0}".format(str(dev_add_response)), "DEBUG")
```

Comment:
self.log("Response from API 'add device' for a single device addition: {0}".format(str(dev_add_response)), "DEBUG")

Reply:
Resolved